docs/works_with_okhttp.md

```diff
@@ -3,14 +3,14 @@ Works with OkHttp
 
 Here’s some libraries that work nicely with OkHttp.
 
- * [Chuck](https://github.com/jgilfelt/chuck): An in-app HTTP inspector for Android OkHttp clients.
+ * [Chucker](https://github.com/ChuckerTeam/chucker): An in-app HTTP inspector for Android OkHttp clients.
+ * [Coil](https://github.com/colinrtwhite/coil): An image loading library for Android backed by Kotlin Coroutines.
```

Comment:
Seems like they moved to `https://github.com/coil-kt/coil`? 🙄

Reply:
Yep! `https://github.com/coil-kt/coil` is the correct URL.

Reply:
Fixed.